server_apps/DB_maintenance/warehouse/chromosomeMartPostgres/refreshChromosomeMart.sql

```diff
@@ -3,30 +3,31 @@
 
 delete from linkage_membership_search_bkup;
```

Comment:
I think we can drop this table all-together.

Reply:
I updated this PR to drop these bkup tables